packages/patternfly-4/react-core/src/components/Dropdown/DropdownWithContext.tsx

```diff
@@ -24,7 +24,7 @@ class DropdownWithContext extends React.Component<DropdownProps & InjectedOuiaPr
     direction: DropdownDirection.down,
     onSelect: (): void => undefined,
     autoFocus: true,
-    ouiaComponentType: 'Dropdown'
+    ouiaComponentType: 'PF4/Dropdown'
```

Comment:
I don't think this is no longer a thing, is it?

Reply:
Right, I missed that.